src/components/EventCard.js

```diff
@@ -1,12 +1,15 @@
 import React, { Fragment } from 'react'
 import PropTypes from 'prop-types'
 
-const EventCard = ({ active, month, url, type, title, date }) => {
+const EventCard = ({ active, month, url, title, date }) => {
+    const icons = ['social', 'workshop', 'talk']
+    const iconClass = icons[Math.floor(Math.random() * icons.length)]
```

Comment:
There's no way to tell the type of event so I'm just randomly showing an icon. Suggestions?

Reply:
At the moment the only 'social' event we are running is 'Casual Catchup'. I suggest making default the icon to be 'talk', then switch to social if title is 'Casual Catchup' and workshop if the the title includes 'Workshop'.

Reply:
Sure, I can do that. Shouldn't be too tricky.